### PR TITLE
Added opset to a folder name

### PR DESCRIPTION
**Pull request by gkrivor**
### Details:
 - Added opset to a folder name

### Tickets:
 - *CVS-95295*

### CODE REVIEW

File: src/tests/functional/plugin/conformance/subgraphs_dumper/src/ops_cache.cpp [hunk 1]
```diff
@@ -159,6 +159,28 @@ float OPCache::get_size_of_cached_ops() {
     return size;
 }
 
+std::string getOpVersion(const ngraph::NodeTypeInfo& type_info) {
```
Comment (iefode) [inline src/tests/functional/plugin/conformance/subgraphs_dumper/src/ops_cache.cpp:162]: Lets move it to funcTestUtils
The lib is linked to subgraphDumper too. The other place where it used is op_summary

Comment (gkrivor) [reply src/tests/functional/plugin/conformance/subgraphs_dumper/src/ops_cache.cpp:162]: ok

**Comment by iefode** [comment]
@gkrivor What is the status of the PR?

File: src/tests/functional/plugin/conformance/subgraphs_dumper/src/ops_cache.cpp [hunk 2]
```diff
@@ -159,6 +159,28 @@ float OPCache::get_size_of_cached_ops() {
...
+        opsets.push_back(ngraph::get_opset2());
+        opsets.push_back(ngraph::get_opset3());
+        opsets.push_back(ngraph::get_opset4());
+        opsets.push_back(ngraph::get_opset5());
+        opsets.push_back(ngraph::get_opset6());
+        opsets.push_back(ngraph::get_opset7());
+        opsets.push_back(ngraph::get_opset8());
+        opsets.push_back(ngraph::get_opset9());
+        opsets.push_back(ngraph::get_opset10());
+    }
```
Comment (ilyachur) [inline src/tests/functional/plugin/conformance/subgraphs_dumper/src/ops_cache.cpp:175]: Please use this method:
https://github.com/openvinotoolkit/openvino/blob/master/src/core/include/openvino/opsets/opset.hpp#L193

**Comment by akladiev** [comment]
This PR will be closed in 2 weeks in case of no activity.

**Pull request closed without merging**